README.md

```diff
@@ -59,6 +59,7 @@ By default, the exporter will try to fetch values from the following commands:
 - `sl.stats` (requires the [SL](http://kamailio.org/docs/modules/stable/modules/sl.html) module)
 - `core.shmmem`
 - `core.uptime`
+- `dlg.stats_active` (requires the [DLG](http://kamailio.org/docs/modules/stable/modules/dialog.html) module)
 
 If you are using the [DISPATCHER](http://kamailio.org/docs/modules/stable/modules/dispatcher.html) module, you can enable `dispatcher.list` as well:
```

Comment:
Maybe you can add a section like this : "If you are using the dialog module..."? What do you think?

Reply:
Yeah, I reformatted it a bit and also added missing info about TLS module. 
I guess a simple table for module -> param would be enough at some point.